packages/manager/modules/support/src/index.js

```diff
@@ -0,0 +1,32 @@
+import '@ovh-ux/manager-core';
+import '@uirouter/angularjs';
+import angular from 'angular';
+import 'oclazyload';
+
+import { state } from './support.routing';
+
+const moduleName = 'ovhManagerSupportLazyLoading';
+
+angular
+  .module(moduleName, [
+    'ovhManagerCore',
+    'oc.lazyLoad',
+    'ui.router',
+  ])
+  .config(/* @ngInject */ ($stateProvider) => {
+    $stateProvider.state(`${state.name}.**`, {
```

Comment:
I find this declaration less explicit that using the state name directly. I need to go to the routing file to get the mapping.

Reply:
We plan to address this in the second wave. Thanks for the inputs!